objectcache/objectcache_interface.go

```diff
@@ -0,0 +1,20 @@
+
+package objectcache
+
+type ObjectCache interface {
```

Comment:
interface should be defined in a different file - `objectcache_interface.go`
Implementation should be in - `objectcache.go`
mocks in `objectcache_mock.go`